src/cpp/server/channelz/channelz_server.cc

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "src/cpp/server/channelz/channelz_server.h"
+
+#include <google/protobuf/text_format.h>
+#include <google/protobuf/util/json_util.h>
+
+#include <grpc/grpc.h>
+#include <grpc/support/alloc.h>
+
+namespace grpc {
+
+ChannelzServer::ChannelzServer() {}
+
+Status ChannelzServer::GetTopChannels(
+    ServerContext* context, const channelz::v1::GetTopChannelsRequest* request,
```

Comment:
Nit: `request` can be a `const &`.

Same below.

Reply:
That is what I thought, but that actually causes this error:

```
/Users/ncteisen/Desktop/grpc/gens/src/proto/grpc/channelz/channelz.grpc.pb.h:201:28: note: hidden overloaded virtual function 'grpc::channelz::v1::Channelz::Service::GetTopChannels' declared here: type
      mismatch at 2nd parameter ('const ::grpc::channelz::v1::GetTopChannelsRequest *' vs 'const ::grpc::channelz::v1::GetTopChannelsRequest &')
    virtual ::grpc::Status GetTopChannels(::grpc::ServerContext* context, const ::grpc::channelz::v1::GetTopChannelsRequest* request, ::grpc::channelz::v1::GetTopChannelsResponse* response);
```